README.md

```diff
@@ -124,8 +124,8 @@ open terminal and run
 rspec spec
```

Comment:
### app/models/user.rb
- [ ] It will be better to validate all the user input (name, email, password, ...) to make sure that anyone with bad intentions cannot compromise their app.

Reply:
### config/routes.rb

- [ ] It would be nice to use only, except, ... options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻

Reply:
### Rspec
- [ ] These 12 failures tests over 24, Please fix this and make sure all tests pass without any errors. 💯 👍🏻 

![image](https://github.com/biruk-bereka/Budget-app/assets/79058364/0d79f96a-cc06-49af-9d44-2867803df4fc)

Reply:
### Add a new transaction page
- [ ] According to the Entity relational database Diagram, there is a many-to-many relationship between Group and Entity so, It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one).  kindly prevent the user to create a new transaction without selecting at least one  categories 

![image](https://github.com/biruk-bereka/Budget-app/assets/79058364/bb97130d-2259-4c82-a70c-56cac6722e00)

Reply:
@BenMukebo here this pages appear only when the user clicks on a category. So I don't the user can make a transaction without going from the specific category.